.gitlab-ci.yml

```diff
@@ -596,11 +596,17 @@ workflow:
   - <<: *if_main_branch
   - <<: *if_release_branch
 
-.on_main_or_release_branch_or_all_builds:
+.on_main_or_release_branch_or_all_builds_or_pkg_installer_changes:
   - <<: *if_main_branch
   - <<: *if_release_branch
   - <<: *if_run_all_builds
   - <<: *if_run_all_unit_tests
+  - changes:
+      paths:
+        - pkg/installer/**/*
+      compare_to: main
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏Usually there is this extra comment when we compare to main
```suggestion
      compare_to: main  # TODO: use a variable, when this is supported https://gitlab.com/gitlab-org/gitlab/-/issues/369916
```